themes/godotengine/pages/education.htm

```diff
@@ -0,0 +1,77 @@
+title = "Education"
+url = "/education"
+layout = "default"
+is_hidden = 0
+==
+{##}
+{% put head_text %}
+<p class="small">
+  Godot is already being used in many classrooms across the world. If you are interested in including it in your
+  curriculum, here you have some answers to the most common questions.
+</p>
+{% endput %}
+
+<style>
+  .question h3 {
+    margin: 2em 0 0;
+  }
+
+  .question p {
+    margin-top: 0.5em;
+  }
+</style>
+
+<h2>FAQ</h2>
```

Comment:
I don't think we need a "FAQ" subheader, as the entire page is a FAQ and that is already explained in the summary at the top of the page.